pkg/operator/hive/hive.go

```diff
@@ -149,7 +149,6 @@ func (r *ReconcileHiveConfig) deployHive(hLog log.FieldLogger, h *resource.Helpe
 		"config/crds/hive_v1_clusterdeployment.yaml",
 		"config/crds/hive_v1_clusterdeprovision.yaml",
 		"config/crds/hive_v1_clusterimageset.yaml",
-		"config/crds/hive_v1_dnsendpoint.yaml",
```

Comment:
Seems like the operator should cleanup this CRD if it exists.

Reply:
OK

Reply:
Looks like no update yet, and conflicts now in place.

Reply:
Yes, no updates yet. This PS has not been a high priority for me. I'll get back to it soon.